ibc_data/ibc_conditions.tsv

```diff
@@ -0,0 +1,379 @@
+Wedge	left_meridian	Visual representation in the left half-plane of the visual field delimited by its vertical meridian
+Wedge	upper_right	Visual representation in the upper-right quadrant of the visual field delimited by its vertical and horizontal meridians
+Wedge	lower_left	Visual representation in the lower-left quadrant of the visual field delimited by its vertical and horizontal meridians
+Ring	foveal	Visual representation in the fovea
+Ring	middle	Visual representation in the mid-periphery of the visual field
+Ring	peripheral	Visual representation in the far-periphery of the visual field
+Raiders	Raiders	Raiders
+Lec2	unattend	A white word is rapidly presented and the participant must ignore it
+Lec2	attend	A black word is rapidly presented and the participant must silently read it to form a short story together with the rest of black words
+Audi	tear	Concatenated sounds of people crying
+Audi	suomi	Suomi speech stimuli
+Audi	yawn	Concatenated sounds of people yawning
+Audi	human	Other human sounds
+Audi	silence	Silence, used as a baseline
+Audi	music	Real-life complex musical sounds
+Audi	reverse	French speech stimuli played in reverse
+Audi	speech	French speech stimuli
+Audi	alphabet	French voice saying the alphabet
+Audi	cough	Concatenated sounds of people coughing
+Audi	environment	Real-life complex environmental sounds
+Audi	laugh	Concatenated sounds of people laughing
+Audi	animals	Real-life animal sounds
+Visu	scrambled	Scrambled image, used as baseline
+Visu	face	Viewing the image of a human face
+Visu	characters	Viewing a string of random characters
+Visu	scene	Viewing the image of a naturalistic scene
+Visu	house	Viewing the image of a house
+Visu	animal	Viewing the image of an animal
+Visu	pseudoword	Viewing a string that conforms a pseudoword
+Visu	tool	Viewing the image of a tool
+Visu	fruit	Viewing the image of a fruit
+Lec1	random_string	A string of random consonants is presented and the participant has to answer if it is all-uppercase or all-lowercase
+Lec1	word	A word is presented and the participant has to decide whether it is a living entity or not
+Lec1	pseudoword	A pseudoword in presented and the participant has to answer whether it has one or two syllabes
+MVEB	letter_occurrence_response	
+MVEB	2_letters_different	The subject must remember 2 characters from a presented string of different letters
+MVEB	4_letters_different	The subject must remember 4 characters from a presented string of different letters
+MVEB	6_letters_different	The subject must remember 6 characters from a presented string of different letters
+MVEB	2_letters_same	The subject must remember 2 characters from a presented string of identical letters
+MVEB	4_letters_same	The subject must remember 4 characters from a presented string of identical letters
+MVEB	6_letters_same	The subject must remember 6 characters from a presented string of identical letters
+MVIS	dot_displacement_response	

```

Comment:
Subject's index finger response, meaning the dot was in the same position than any of the previously presented ones.

Reply:
Same as line 151